cfme/networks/views.py

```diff
@@ -82,12 +82,15 @@ class NetworkProviderDetailsView(BaseLoggedInPage):
     @View.nested
     class entities(View):  # noqa
         """ Represents details page when it's switched to Summary/Table view """
+        # TODO: remove all SummaryTables in favor of ParametrizedSummaryTable when safe
         properties = SummaryTable(title="Properties")
         relationships = SummaryTable(title="Relationships")
         status = SummaryTable(title="Status")
         overview = SummaryTable(title="Overview")
         smart_management = SummaryTable(title="Smart Management")
 
+        summary = ParametrizedView.nested(ParametrizedSummaryTable)
```

Comment:
Why is this a nested ParametrizedView?  I think just setting it as an instance of `ParametrizedSummaryTable()` is the correct thing to do here, see existing uses of it in the codebase.

Reply:
You're right, fixed. It works either way so I didn't even notice 😄